app/models/cloud_volume.rb

```diff
@@ -18,6 +19,14 @@ class CloudVolume < ApplicationRecord
 
   acts_as_miq_taggable
 
+  supports :cloud_volume_backup_create do
+    unsupported_reason_add(:cloud_volume_backup_create, _("Backup unsupported")) unless validate_backup_create
```

Comment:
can `validate_backup_create` be actually be inlined to a condition that checks why it's supporte or not?
This way it just returns false or true for no apparent reason